loopchain/peer/block_manager.py

```diff
@@ -608,7 +610,7 @@ def __block_request_to_peers_in_sync(self, peer_stubs, my_height, unconfirmed_bl
                             util.exit_and_msg(f"This peer already tried to synchronize {my_height} block "
                                               f"for max retry number({conf.BLOCK_SYNC_RETRY_NUMBER}). "
                                               f"Peer will be down.")
-            else:
+            elif response_code == message_code.Response.fail:
```

Comment:
Then when`response_code == message_code.Response.fail_no_confirm_info` case, shouldn't it retry sync to another target? I guess there's no changing of `peer_index` when case of no_confirm_info, so it might retry to same target again and again.